examples/run_standalone_test.cmake

```diff
@@ -0,0 +1,31 @@
+# run_standalone_test.cmake
+
+set(STANDALONE_SOURCE_DIR "${PROJECT_CURRENT_SOURCE_DIR}/standalone_project")
+set(STANDALONE_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/standalone_build")
+
+file(MAKE_DIRECTORY "${STANDALONE_BINARY_DIR}")
+
+execute_process(
+  COMMAND ${CMAKE_COMMAND} -S "${STANDALONE_SOURCE_DIR}" -B "${STANDALONE_BINARY_DIR}"
+  RESULT_VARIABLE result
+)
```

Comment:
can you add a brief comment on what this does (same below)? could this be `add_subdirectory` or so?

Reply:
`add_subdirectory` will not work here. This is essentially the "test script" to run when `ctest` is invoked, including configuring, building, and running tests.